server/src/server/env-config.js

```diff
@@ -17,6 +17,7 @@ const config = {
     server: {
         port: process.env.PORT || 3000,
         host: process.env.HOST || 'localhost',
+        jwtSecret: process.env.INTERNAL_JWT_SECRET,
```

Comment:
```suggestion
        jwtSecret: getRequiredEnv(process.env.INTERNAL_JWT_SECRET),
```

Reply:
This is technically not required as you could use `sendmail` in production